### PR TITLE
Add totk presets

### PR DESCRIPTION
**Pull request by savage13**
Towers, Shrines, Lightroots, Koroks, Caves, Well, Chasms, Bubbulfrogs

### CODE REVIEW

**Comment by Pistonight** [comment]
Thanks for the PR. Just a couple questions：

1. Where are the referenced icons?
2. I will probably make some tweaks to it, like extracting the data and using a script to generate the preset files. Would that be fine?

**Comment by savage13** [comment]
- I was unsure about the icons.
- I started building a basic html page to generate the presets, built on javascript and running queries on a database.  The major issue is getting the UI Names from the database.  I modified the database to output the ui names.

**Comment by savage13** [comment]
I did the PR directly on the website cause it was "easier".  I can add the icons later in a separate PR if desired.

File: presets/totk/chasms.yaml [hunk 1]
```diff
@@ -0,0 +1,185 @@
+icons:
+  chasm:
+    use: ./chasm.png
```
Comment (Pistonight) [inline presets/totk/chasms.yaml:3]: Should this be cave?

Comment (Pistonight) [reply presets/totk/chasms.yaml:3]: My bad, the commend is supposed to be for line 5

**Comment by Pistonight** [comment]
I think some of the presets like caves and chasms can benefit from #30, where people (I) might want to have it in a movement config and also have an icon show up on the map, instead of having a line in the doc that says X cave.

I am fine with the presets. Please add icons as well otherwise the presets won't work

**Review by Pistonight — APPROVED** [review]

**Pull request merged**